db/active_replicator_checkpointer.go

```diff
@@ -53,7 +53,21 @@ func NewCheckpointer(ctx context.Context, clientID string, blipSender *blip.Send
 	}
 }
 
-func (c *Checkpointer) ProcessedSeq(seq string) {
+func (c *Checkpointer) AddIgnoredSeq(seq string) {
```

Comment:
Although it's a bit more wordy, I think AlreadyKnown is a lot more self-descriptive than 'Ignored' everywhere it's being used in this PR.  Ignored didn't immediately register for me as 'change sent that the target already had'.

Reply:
Had trouble naming this between ignored and skipped, but I agree 'AlreadyKnown' is a way clearer option.